daisy/image.go

```diff
@@ -139,9 +158,20 @@ func (i *Image) populate(ctx context.Context, s *Step) dErr {
 		}
 	}
 	i.link = fmt.Sprintf("projects/%s/global/images/%s", i.Project, i.Name)
+	i.populateGuestOSFeatures(s.w)
 	return errs
 }
 
+func (i *Image) populateGuestOSFeatures(w *Workflow) {
```

Comment:
Any validation needed for guest os features?

Reply:
Sadly I can't easily validate features without hardcoding them into our code.